endpoints/books/controller.py

```diff
@@ -33,38 +36,49 @@ def post(self):
         try:
             current_user = get_jwt_identity()
             book = request.get_json()
-            book['user_id'] = current_user
+            existing_book = userBookRepository.findByName(book['name'])
+            if existing_book['name'] == book['name'] and existing_book['user_id'] == current_user['id']:
```

Comment:
Question, if you're storing saved books, shouldn't you keep this in a separate table as a relationship between books and users?. Also why do you need to do `existing_book['name'] == book['name']` if you already get the proper book name in the previous query

Reply:
I'm just checking if the user isn't adding a resource with the same name,  I'll work on that statement.